src/java.base/share/classes/jdk/internal/foreign/abi/AbstractLinker.java

```diff
@@ -37,6 +37,7 @@
 import jdk.internal.foreign.abi.aarch64.windows.WindowsAArch64Linker;
 import jdk.internal.foreign.abi.ppc64.aix.AixPPC64Linker;
 import jdk.internal.foreign.abi.ppc64.sysv.SysVPPC64leLinker;
+import jdk.internal.foreign.abi.fallback.FallbackLinker;
```

Comment:
Please adjust imports so additions for OpenJ9 keep things ordered.

Reply:
Fixed as suggested above.